packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -104,7 +103,7 @@ const defaultProps = {
   placeholder: undefined,
   required: false,
   rows: null,
-  size: 'small',
+  size: undefined,
```

Comment:
Changing the default props is a non-passive change. This will require a major version bump of the component if we want to continue with this solution.

Reply:
Added prop in fe94213